gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java

```diff
@@ -145,6 +145,10 @@ protected LabelProvider createLabelProvider(Theme theme) {
   }
 
   public void setInput(T root) {
+    // Clear the selection, since we handle maintaining the selection ourselves and so
+    // don't want JFace's selection preserving, as it appears to be broken on input
+    // change (see https://github.com/google/gapid/issues/1264)
+    setSelection(null);
```

Comment:
Does this mean we lose the current state selection switching commands, command selection switching context, etc?

Reply:
No, we maintain the selection ourselves. We need to, because the tree completely changes. JFace's attempt to preserve the selection should fail, since it won't find the selection in the new tree (and usually doesn't). 

To put it another way, this change doesn't make us lose the selection any more or less than we do now.

Reply:
Cool. Thanks